README.md

```diff
@@ -1,24 +1,150 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
 
-* Ruby version
+# 📖 [hello rails back end] <a name="about-project"></a>
 
-* System dependencies
+**[hello rails back end]** is an app that shows you a random greeting on page refresh.
 
-* Configuration
+## Link to front end repo
 
-* Database creation
+- [] [React Front End](https://github.com/martinkarugaba/hello_react_front_end)
```

Comment:
- [ Optional ] Kindly I recommend you  to update the link to React front end to the feature branch with the front end project, Currently  the link redirect to dev branch , Kindly update the link.